views_api.py

```diff
@@ -182,17 +181,6 @@ async def api_check_fiat_rate(currency):
     return {"rate": rate}
 
 
-@lnurlp_ext.delete("/api/v1", dependencies=[Depends(check_admin)])
-async def api_stop():
```

Comment:
- the endpoint is being removed without a `lnurlp_stop` being added

Reply:
damn, yes you are so right, still important for dynamic stopping, i was just thinking about restarting

Reply:
done